src/widget/wrecordingduration.h

```diff
@@ -1,6 +1,6 @@
 // wrecordingduration.h
 // WRecordingDuration is a widget showing the duration of running recoding
-// In skin.xml, it is represented by a <RecordingDuration> node.
+// In skin it is represented by a <RecordingDuration> node.
```

Comment:
How about "In skin XML, it is represented by a <RecordingDuration> node."

Reply:
Sure, I don't mind